test/test_framework.py

```diff
@@ -612,6 +612,110 @@ class CustomObject:
 
             framework_copy.commit()
 
+    def test_comparison_operations(self):
+        class SomeObject(Object):
+            state = StoredState()
+
+            def __init__(self, framework, key):
+                super().__init__(framework, key)
+
+        test_operations = [(
+            {"1"},  # Left operand.
+            {"1", "2"},  # Right operand.
+            lambda a, b: a < b,  # An operation to test.
+            True,  # The expected result of operation(left_operand, right_operand).
+            False,  # The expected result of operation(right_operand, left_operand).
```

Comment:
It stops making sense to call them Left and Right if we use Right on the Left.

This is both terse and pleasant:
```
# Operand A.
# Operand B.
# Operation to test.
# Result of op(A, B)
# Result of op(B, A)
```

Reply:
Note that aligning them as above would make it nicer to read as well.